test/fixtures/semantic-html5-scenarios/section.html

```diff
@@ -0,0 +1,6 @@
+<section id="_section_title">
+<h2>Section Title</h2>
+<p>
+A normal paragraph.
+</p>
```

Comment:
I know this isn't part of this PR, but I don't like that there are newlines directly inside the `<p>` tag. This strikes me as something people are going to complain about as being weird HTML, which is what we're trying to get away from. Should I file a separate issue?

Reply:
Yes sure 👍🏻

Reply:
Done: https://github.com/asciidoctor/asciidoctor/issues/4057